### ISSUE TITLE
squatter's usage() and manual page don't list all allowed options

### ISSUE THREAD

**Issue by rsto** [cyrusimap/cyrus-imapd#1826]
squatter accepts options ````C:I:N:RXT:S:Fc:de:f:mn:rsiavz:t:ou```` but both it's
usage() message as well as the manual page are missing a couple of these

**Comment by postilion** [comment]
@rsto, I'll be happy to take care of the manpage updates, as long as the usage message is corrected.

**Comment by rsto** [comment]
I've completely overhauled the usage message in https://github.com/cyrusimap/cyrus-imapd/commit/4643639d6fc9637b964fa1cc6090ec40cfbc099b

Note that getopt was instructed to accept option 's' but squatter rejected it as unknown. I've removed it. I also started to cross-check the usage string with the manual page and it seems as if we need to iterate: Some options in the manual page are not supported by squatter (-A comes to mind), others aren't documented.

Is the new usage() message OK for you to read? It's a bit tricky to show on the command line which options are OK to combine with which one. Thanks.
